### PR TITLE
[ceph_ansible] new plugin: collect ceph-ansible site.yml and group_vars

### PR DESCRIPTION
**Pull request by haklein**
Signed-off-by: Harald Klein <hari@vt100.at>

### CODE REVIEW

**Comment by squizzi** [comment]
Harald and are going to combine our work into this existing PR and I will close out #927 and he will attribute me in his commit.

File: sos/plugins/ceph_ansible.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    packages = (
+        'ceph-ansible',
+    )
+
+    def setup(self):
+        self.add_copy_spec([
+            "/usr/share/ceph-ansible/group_vars/",
+            "/usr/share/ceph-ansible/site*.yml",
+            "/usr/share/ansible/ansible.cfg"
```
Comment (bmr-cymru) [inline sos/plugins/ceph_ansible.py]: Doesn't this file belong to `ansible` rather than to `ceph-ansible`?

Comment (haklein) [inline sos/plugins/ceph_ansible.py]: thanks bmr that's been a typo taken over from PR #927. Just fixed and forcibly updated.

Comment (bmr-cymru) [inline sos/plugins/ceph_ansible.py]: OK, well, it's vanished in the latest version (or turned into `/usr/share/ceph-ansible/ansible.cfg`).

**Pull request closed without merging**